src/status_im2/contexts/chat/composer/sub_view.cljs

```diff
@@ -47,7 +47,8 @@
          :label               (i18n/label :t/jump-to)
          :style               {:align-self :center}}}
        {}]]
-     (when @show-floating-scroll-down-button?
+     (when (and (not @focused?)
+                @show-floating-scroll-down-button?)
```

Comment:
Two things:

1. Should the scroll down button not be visible at all when focused?
2. If so, we can update `@show-floating-scroll-down-button?` to not be true unless the composer is not foucsed

Reply:
1 yes
2 okay good idea will do

Reply:
Okay, When I started to implement the solution I realised that I'd have to get the composer state in the messages list state or vice versa, Which I don't think is a good idea.
We're going to have to pass the state of the composer (Where it's focused or not) to the messages list logic
What do you think about it?

Reply:
You can get if the input is focused using this subscription: `(:focused? (rf/sub [:chats/current-chat-input]))`

Reply:
Nice, Thanks

Reply:
Done, I couldn't use that subscription inside the on-scroll because it's executed only on scroll.
But I did it using another approach.

Reply:
@OmarBasem Kindly take another look.